### DISCUSSION TITLE
Contact Form Submission - Suggestion

### DISCUSSION THREAD

**Discussion by maggieliu05** [cpinitiative/usaco-guide#2674] [Ideas]
Someone submitted the contact form!

**URL**: https://usaco.guide/gold/dp-bitmasks?lang=cpp
**Module**: Gold - Bitmask DP
**Topic**: Suggestion
**Message**: 
Problem "Uddered but not Herd" difficulty is definitely not "Easy". I would say "Hard" or even "Very Hard".

**Comment by SansPapyrus683** [comment]
how bout "normal" instead?
